dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnectorTest.java

```diff
@@ -51,16 +51,19 @@
 import org.junit.Assert;
 import org.junit.Assume;
 import org.junit.Test;
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.FromDataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
```

Comment:
Optional: This comment can be removed. It doesn't document the class, it only tells the reader how to use gradle.

Reply:
I find it helpful, because it provides the full command for using Java 8, so I would prefer to keep it. However, since the constraint about Java version applies to a single test only, I've moved the comment to this single test method.